src/server.rs

```diff
@@ -1226,62 +1227,57 @@ where
         let span = self.span.clone(); // XXX(eliza): T_T
```

Comment:
it could be worth seeing if we can remove the `clone` call now --- IIRC, this was necessary because we borrowed all of `self` previously, which may no longer be the case here?

Reply:
sorry, i was wrong about this, looks like this broke the build: https://github.com/hyperium/h2/runs/4884456490?check_suite_focus=true